### ISSUE TITLE
Alert is not read in Chrome

### ISSUE THREAD

**Issue by aprematta** [FreedomScientific/VFO-standards-support#562]
### Summary

When an alert occurs on a web page in Chrome, JAWS does not read the contents.

Example:
1. Go to https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_alert
2. Select the Try it button

### Expected result

JAWS would say 'An Embedded page on this page says 'Hello! I am an alert box'.....OK button'

### Actual result

JAWS says 'An Embedded page on this page says.....OK button'

### Additional Information
This was working on a previous instance of JAWS. Once upgraded to the latest version it stopped.

#### JAWS version and build number
2021.2107.12 ILM

#### Browser and version:
Chrome Version 92.0.4515.131

**Comment by JAWS-test** [comment]
Duplicate of https://github.com/FreedomScientific/VFO-standards-support/issues/351and https://github.com/FreedomScientific/VFO-standards-support/issues/361

**Comment by menovak** [comment]
Per Duplicate of #351 and #361, so outcome would be the same....
